### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by jokay**
Fix typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ Cake AddIn that extends Cake with Git features using LibGit2 and LibGit2Sharp.
 Documentation is available at [cakebuild.net/dsl/git](http://cakebuild.net/dsl/git).
 
 ## Example usage
-```cake
+
+```csharp
```
Comment (gep13) [inline README.md]: @x-jokay is there a reason that this was changed?  To the best of my knowledge, `cake` is a supported language type on GitHub.  Perhaps @devlead can confirm.

**Review by gep13 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (devlead) [reply README.md]: Yes `cake` is an supported GH flavored markdown code fence language
image

Comment (x-jokay) [reply README.md]: Good to know :)

**Comment by x-jokay** [comment]
@gep13 ready for review.

**Review by gep13 — APPROVED** [review]
LGTM!

**Review by pascalberger — APPROVED** [review]

**Comment by pascalberger** [comment]
@x-jokay your changes have been merged, thanks for your contribution 👍

**Pull request merged**